### PR TITLE
Modify managing-database-transactions module.md file

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Modify managing-database-transactions module.md file and update templateDetails.json file accordingly.

### CODE REVIEW

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/src/managing_database_transactions/Module.md [hunk 1]
```diff
@@ -1,159 +1,56 @@
...
-INFO  [managing_database_transactions] - Verifying whether account ID 2 exists
-INFO  [managing_database_transactions] - Depositing money to account ID: 2
-INFO  [managing_database_transactions] - $300 has been deposited to account ID 2
-INFO  [managing_database_transactions] - Transaction committed
-INFO  [managing_database_transactions] - Successfully transferred $300 from account ID 1 to account ID
+$ ballerina build --experimental managing_database_transactions 
 ```
 
-- For `Scenario 2` where 'Alice' tries to transfer $500 to Bob's account, the transaction is expected to fail as 'Alice' has an insufficient balance
+This creates the executables. Now run the `managing_database_transactions.jar` file created in the above step.Path to the `ballerina.conf` could be provided using the `--b7a.config.file` option.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/src/managing_database_transactions/Module.md]:
```suggestion
This creates the executables. Now, run the `managing_database_transactions.jar` file created in the above step. Path to the `ballerina.conf` file can be provided using the `--b7a.config.file` option.
```

Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/src/managing_database_transactions/Module.md]: Fixed

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md [hunk 2]
```diff
@@ -66,7 +65,23 @@ managing-database-transactions
             ├── account_manager_test.bal
             └── resources
 ```
-Open the project with VS Code and write the integration implementation and tests in the `account_manager.bal`, `application.bal`, and `account_manager_test.bal` files respectively.
+
+Create a folder called `lib` under the project root path. Copy the [JDBC driver for MySQL](https://dev.mysql.com/downloads/connector/j/) into the `lib` folder.
+
+#### 3. Add project configurations file
+Add a `ballerina.conf` file and create .bal files with meaningful names as shown in the project structure given below.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]: Project structure is given above right?

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md [hunk 3]
```diff
@@ -66,7 +65,23 @@ managing-database-transactions
...
+
+#### 4. Write the integration
+
+Open the project with VS Code. The integration implementation is written in the `src/managing_database_transaction/account_manager.bal` and `src/managing_database_transaction/application.bal`. files
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]:
```suggestion
Open the project with VS Code. The integration implementation is written in the `src/managing_database_transaction/account_manager.bal` and `src/managing_database_transaction/application.bal`. files.
```

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md [hunk 4]
```diff
@@ -66,7 +65,23 @@ managing-database-transactions
...
+file.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]: Delete.

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md (hunk 2 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]: Fixed

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md (hunk 3 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]: Fixed

File: docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md (hunk 4 above)
Comment (Nipunich) [reply docs/content/src/learn/tutorials/database-integrations/managing-database-transactions/README.md]: Fixed

**Pull request merged**